streaming/base/storage/download.py

```diff
@@ -322,6 +322,30 @@ def download_from_azure_datalake(remote: str, local: str) -> None:
         raise
 
 
+def download_from_databricks_unity_catalog(remote: str, local: str) -> None:
+    """Download a file from remote Databricks Unity Catalog to local.
+
+    Args:
+        remote (str): Remote path (Databricks Unity Catalog).
+        local (str): Local path (local filesystem).
+    """
+    from databricks.sdk import WorkspaceClient
+
+    obj = urllib.parse.urlparse(remote)
+    if obj.scheme != 'uc':
+        raise ValueError(
+            f'Expected obj.scheme to be `uc`, instead, got {obj.scheme} for remote={remote}')
+
+    client = WorkspaceClient()
+    file_path = remote.lstrip('uc:/')
+    uc_file = client.files.download(file_path)
```

Comment:
Wondering, does `client.files.download()` method support taking `destination file path` as an argument apart from `source file path` which in our case is `file_path` ?

Reply:
No, it only takes one arg: https://github.com/databricks/databricks-sdk-py/blob/721a2ac17d4732724a30aa66384185ae0ff11ae3/databricks/sdk/mixins/files.py#L410